tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -518,7 +518,7 @@ When('the user picks the row of file/folder {string} in the webUI', function(ite
 })
 
 When('the user switches to {string} tab in details panel using the webUI', function(tab) {
```

Comment:
hm.... would be nice to rename `tab` in step as well. If that doesn't ripple too far.